test/WebJobs.Script.Tests/Description/FunctionInvokerBaseTests.cs

```diff
@@ -111,8 +124,8 @@ public void LogInvocationMetrics_EmitsExpectedEvents()
             var completedStartEvent = (FunctionStartedEvent)_metricsLogger.MetricEventsEnded[0];
             Assert.Same(startedEvent, completedStartEvent);
             Assert.True(completedStartEvent.Success);
-            var message = _traceWriter.Traces.Last().Message;
-            Assert.True(Regex.IsMatch(message, $"Function completed \\(Success, Id={executionContext.InvocationId}, Duration=[0-9]*ms\\)"));
+            //var message = _testLoggerProvider.GetAllLogMessages().Last().FormattedMessage;
+            //Assert.True(Regex.IsMatch(message, $"Function completed \\(Success, Id={executionContext.InvocationId}, Duration=[0-9]*ms\\)"));
```

Comment:
You should be able to remove these. The invoker doesn't do any logging of 'started'/'completed' anymore. That's all handled by the SDK layer. Originally it was done here so we could redirect these logs to the correct function log file, but with the switch to use ILogger exclusively, that's already taken care of. When the `FileLogger` sees that the 'executed'/'executing' logs have the `Function.{FunctionName}` category, it knows to route them to the correct file. Since those logs were duplicates of what we were already doing, I pulled them out.

Reply:
Awesome. Thanks for the context. Was driving me mad. :)